hoverkite-protocol/src/response.rs

```diff
@@ -1,27 +1,26 @@
 use crate::util::{ascii_to_bool, bool_to_ascii};
-use crate::{ParseError, Side};
+#[cfg(feature = "std")]
+use crate::WriteCompat;
+use crate::{ProtocolError, Side};
+use arrayvec::ArrayString;
 use core::convert::TryInto;
 use core::mem::size_of;
 use nb::Error::{Other, WouldBlock};
-#[cfg(feature = "std")]
-use std::collections::VecDeque;
+const BUF_SIZE: usize = 256;
```

Comment:
`MAX_LOG_SIZE` might be a more descriptive name. And add a blank line between it and the `use` statements.

Reply:
I'd still like a blank line between the `use` statements and the `const`.